config/config.yaml

```diff
@@ -31,6 +31,10 @@ tls:
   cert_file: /path/to/cert
   key_file: /path/to/key
 
+# Domains to be able to use virtual-hosted-style access to bucket.
+listen_domains:
+  0: s3dev.neofs.devenv
```

Comment:
What do you think about making it an array in this PR? 
Don't think we benefit from having `0`,`1`,`2`..., prefix there. Some old stuff we are trying to avoid now.